docs/release_guide.rst

```diff
@@ -42,3 +60,57 @@ You probably want to make sure (with ``git tag``) that the new version
 number is available. If any release activities fail while running this
 command, you may safely re-run this command. You can also safely undo
 previously run activities. Please see the rever docs for more details.
+
+..
+    I'm guessing we want this to just push the tag to GitHub.
+    Things that could differ from matplotlib in this step
+    * Empty commit
```

Comment:
The empty commit is because Matplotlib uses versioneer to inject the version which injects the names of all of the references to the tagged commit into the tarball so if you have both the tag and a branch and then later just a tag the hash of the tarball will change.